pages/common/hsd-cli.md

```diff
@@ -0,0 +1,36 @@
+# hsd-cli
+
+> The command line REST tool for the Handshake blockchain.
+> Homepage: <https://handshake.org>.
+
+- Retrieve information about the current server:
+
+`hsd-cli info`
+
+- Broadcast a local transaction:
+
+`hsd-cli broadcast {{transaction_hex}}`
+
+- Retrieve a mempool snapshot:
+
+`hsd-cli mempool`
```

Comment:
Is there any way of specifying which snapshot you want, or is ti automatic?

Reply:
You basically get a JSON array of all transactions in the mempool at the current time.

Reply:
Ahhh ok :-)